docs/reference/api/connectivity/networksocket/UDPSocket.md

```diff
@@ -2,6 +2,14 @@
 
 The UDPSocket class provides the ability to send packets of data over UDP, using the `sendto` and `recvfrom` member functions. Packets can be lost or arrive out of order, so we suggest using a [TCPSocket](/docs/v5.6/reference/tcpsocket.html) when you require guaranteed delivery.
 
+The constructor takes in the NetworkStack pointer to open the socket on the specified NetworkInterface. If NetworkStack is not passed in the constructor, then you must call `open` to initialize the socket.
```

Comment:
Query: Who or what passes in the constructor?

Reply:
NetworkStack pointer